tests/flows/test_sklearn.py

```diff
@@ -0,0 +1,459 @@
+from collections import OrderedDict
+import json
+import os
+import sys
+import unittest
+
+import numpy as np
+import scipy.optimize
+import scipy.stats
+import sklearn.base
+import sklearn.datasets
+import sklearn.decomposition
+import sklearn.dummy
+import sklearn.ensemble
+import sklearn.feature_selection
+import sklearn.gaussian_process
+import sklearn.model_selection
+import sklearn.pipeline
+import sklearn.preprocessing
+import sklearn.tree
+
+from openml.flows import OpenMLFlow, sklearn_to_flow, flow_to_sklearn
+
+
+class Model(sklearn.base.BaseEstimator):
+    def __init__(self, boolean, integer, floating_point_value):
+        self.boolean = boolean
+        self.integer = integer
+        self.floating_point_value = floating_point_value
+
+    def fit(self, X, y):
+        pass
+
+
+class TestSklearn(unittest.TestCase):
+    
+    def setUp(self):
+        iris = sklearn.datasets.load_iris()
+        self.X = iris.data
+        self.y = iris.target
+
+    def test_serialize_model(self):
+        model = sklearn.tree.DecisionTreeClassifier(criterion='entropy',
+                                                    max_features='auto',
+                                                    max_leaf_nodes=2000)
+
+        fixture_name = 'sklearn.tree.tree.DecisionTreeClassifier'
+        fixture_description = 'Automatically created sub-component.'
+        fixture_parameters = \
+            OrderedDict((('class_weight', 'null'),
+                         ('criterion', '"entropy"'),
+                         ('max_depth', 'null'),
+                         ('max_features', '"auto"'),
+                         ('max_leaf_nodes', '2000'),
+                         ('min_impurity_split', '1e-07'),
+                         ('min_samples_leaf', '1'),
+                         ('min_samples_split', '2'),
+                         ('min_weight_fraction_leaf', '0.0'),
+                         ('presort', 'false'),
+                         ('random_state', 'null'),
+                         ('splitter', '"best"')))
+
+        serialization = sklearn_to_flow(model)
+
+        self.assertEqual(serialization.name, fixture_name)
+        self.assertEqual(serialization.class_name, fixture_name)
+        self.assertEqual(serialization.description, fixture_description)
```

Comment:
This is a default value that is added to each component. Issue #175 is about changing this in a useful manner; so I will skip this for now to not complicate this PR further.